lib/gateway/Shard.js

```diff
@@ -2304,6 +2336,14 @@ class Shard extends EventEmitter {
                 }
                 channel.guild.threads.add(channel, this.client);
                 this.client.threadGuildMap[packet.d.id] = packet.d.guild_id;
+
+                const parent = channel.guild.channels.find((c) => c.id == channel.parentID);
+
+                if(parent) {
+                    parent.last_message_id = channel.id;
```

Comment:
This should only cover forum channels, for other channel types this should be left as actual messages in the text channel

Reply:
Would you prefer `parent.type === 15` or `parent instanceof ForumChannel`?

Reply:
Latter for consistency with areas with similar behaviour

Reply:
Gotcha 👍🏻